.gitignore

```diff
@@ -1,2 +1,3 @@
 /pkg/
+/Gemfile.lock
```

Comment:
This is the `.gitignore`, the change was made in [0ed2f74](https://github.com/premailer/css_parser/commit/0ed2f74e8d6e6ba26c4260aaf4cb4c57159231ec). I'm going to take a closer look though, I'm not as familiar with the way that Github shows diffs, I'm surprised it's showing this and other changes as changes when they currently match the master.